src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -32,7 +32,8 @@ public class ModelManager implements Model {
     private State state = State.SCHEDULE; // Default state of app. Can be either SCHEDULE or STUDENTS
     private Person currentShowingPerson = null;
     private Lesson currentShowingLesson = null;
-    private Task currentEditingLesson = null;
```

Comment:
was this a typo from last time where currentEditingLesson is a Task type?

Reply:
Sorry, it is a typo, thank you for pointing that out.

Reply:
I think it was a typo lastTime; I renamed it to currentShowingTask.